assets/js/modules/analytics-4/utils/tag-matchers.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Tag matching patterns.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const tagMatchers = [
+	// Detect gtag script calls.
+	/<script [^>]*src=['|"]https:\/\/www.googletagmanager.com\/gtag\/js\?id=G-([a-zA-Z0-9]+)['|"][^>]*><\/script>/i,
```

Comment:
Technically the unescaped dots here will work, but will arguably match things it shouldn't. Probably won't ever matter, but let's do:

```suggestion
	/<script [^>]*src=['|"]https:\/\/www\.googletagmanager\.com\/gtag\/js\?id=G-([a-zA-Z0-9]+)['|"][^>]*><\/script>/i,
```

Reply:
Yep, agree.